frontend/lib/src/components/widgets/DataFrame/hooks/useTableSizer.ts

```diff
@@ -53,15 +35,14 @@ export type AutoSizerReturn = {
   minWidth: number
   // The maximum width of the data grid can be resized to
   maxWidth: number
+  // The row height of the data grid
+  rowHeight: number
   // The current (or initial) size of the data grid
   resizableSize: ResizableSize
   // A callback function to change the size of the data grid.
   setResizableSize: React.Dispatch<React.SetStateAction<ResizableSize>>
 }
 
-export function calculateMaxHeight(numRows: number): number {
```

Comment:
nit: does this make sense to remove and then duplicate a couple of times in the code & tests?

Reply:
It was only used once in the app but a couple of times in the test. But with the change to `rem`, it's a lot harder to reuse this method in the tests since the root font size is not set up properly at the point where we used it. The calculation also isn't correct anymore since eventually, we will allow data rows and column headers to have different sizes (via config). So, we cannot just do numRows * ROW_HEIGHT anymore it would need to be more adapted to the specific situation.

Reply:
Oh ok, thanks for the explanation